CONTRIBUTING.md

```diff
@@ -103,11 +103,7 @@ If deploying the site to a production environment, make sure to minify the JS fi
 ### Styleguide
 
 ```sh
-npm install --dev
-npm run init-styleguide
-cd styleguide-template && npm install
-cd ..
-npm run watch-styleguide
+docker-compose run webpack npm run build-styleguide
```

Comment:
Do you want to also include some information about how to run the styleguide in watch mode instead? It seems like you'd want to be doing watch mode during dev and build would really only happen on Federalist?